integrations/airflow/prometheus_metadata.yaml

```diff
@@ -0,0 +1,40 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/airflow_scheduler_tasks_running/gauge
+    exporter_metadata:
+      name: Airflow Prometheus Exporter
+      doc_url: https://github.com/prometheus/mysqld_exporter
+      minimum_supported_version: "2.3.0"
```

Comment:
Update the doc_url here. And is there a minimum supported version applicable?

Reply:
Doc url was wrong, but version is correct.